### PR TITLE
[1.18] Vendor: bump github.com/containers/libpod from v1.8.0 to v1.8.1

### PR DESCRIPTION
**Pull request by mccv1r0**
Vendor: bump github.com/containers/libpod from v1.8.0 to v1.8.1

Signed-off-by: Michael Cambria <mcambria@redhat.com>

/kind feature

bump libpod vendor code from v1.8.0 to v1.8.1 to obtain updated libpod and cni features

Fixes None

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,20 +9,24 @@ require (
...
+	github.com/containers/storage v1.16.2
+	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f // indirect
 	github.com/coreos/go-systemd/v22 v22.0.0
 	github.com/cpuguy83/go-md2man v1.0.10
 	github.com/cri-o/ocicni v0.1.1-0.20191204151446-e25b504ca708
 	github.com/docker/docker v1.4.2-0.20200203170920-46ec8731fbce
 	github.com/docker/go-units v0.4.0
+	github.com/docker/spdystream v0.0.0-20181023171402-6480d4af844c // indirect
+	github.com/elazarl/goproxy v0.0.0-20190421051319-9d40249d3c2f // indirect
+	github.com/elazarl/goproxy/ext v0.0.0-20190911111923-ecfe977594f1 // indirect
```
Comment (saschagrunert) [inline go.mod]: Which go version do you use for vendoring? I think those indirect dependencies are not needed to be added any more in go 1.14.

Comment (mccv1r0) [reply go.mod]: I used go v1.13

Comment (haircommander) [reply go.mod]: this is release-1.18, which is still on go 1.13

**Comment by haircommander** [comment]
'Error response from daemon: manifest for quay.io/crio/crio-build-386-go1.13:1.18-1.2.0 not found'

reproduced locally, not the fault of this PR, let me try to repush it

**Comment by haircommander** [comment]
/lgtm

(needs approval @saschagrunert @mrunalp @umohnani8 @rhatdan )

File: go.mod (hunk 1 above)
Comment (mccv1r0) [reply go.mod]: Using go v1.14 in my sb doesn't change anything:

`
[mcambria@mcambria cri-o]$ git branch -vv
  libpodVendor181                  e02dd7ead176 [master: behind 1] vendor: bump github.com/containers/libpod from v1.8.0 to v1.8.1
  master                           eae3c150134e [origin/master: ahead 2] Merge pull request #3401 from mccv1r0/libpodVendor181
* release-1.18                     40749edb92f2 [upstream/release-1.18: ahead 1] Vendor: bump github.com/containers/libpod from v1.8.0 to v1.8.1
[mcambria@mcambria cri-o]$ 
[mcambria@mcambria cri-o]$ 
[mcambria@mcambria cri-o]$ git status 
On branch release-1.18
Your branch is ahead of 'upstream/release-1.18' by 1 commit.
  (use "git push" to publish your local commits)

nothing to commit, working tree clean
[mcambria@mcambria cri-o]$ podman run --privileged --rm --env HOME=/root -v `pwd`:/src -w /src docker.io/library/golang:1.14 make vendor 
export GO111MODULE=on \
        go mod tidy && \
        go mod vendor && \
        go mod verify
go: downloading github.com/onsi/ginkgo v1.12.0
go: downloading k8s.io/klog v1.0.0
[deleted]
go: downloading github.com/containerd/cgroups v0.0.0-20190919134610-bf292b21730f
go: downloading github.com/xeipuuv/gojsonpointer v0.0.0-20190809123943-df4f5c81cb3b
all modules verified
[mcambria@mcambria cri-o]$ 
[mcambria@mcambria cri-o]$ git status 
On branch release-1.18
Your branch is ahead of 'upstream/release-1.18' by 1 commit.
  (use "git push" to publish your local commits)

nothing to commit, working tree clean
[mcambria@mcambria cri-o]$ 

`

**Pull request merged**